src/libNode/Node.cpp

```diff
@@ -1655,7 +1656,7 @@ void Node::RejoinAsNormal(bool rejoiningAfterRecover) {
         }
         BlockStorage::GetBlockStorage().RefreshAll();
         AccountStore::GetInstance().RefreshDB();
-        if (this->Install(SyncType::NORMAL_SYNC, true, rejoiningAfterRecover)) {
+        if (this->Install(SyncType::NORMAL_SYNC, true, true)) {
```

Comment:
Why rejoiningAfterRecover becomes always true here?

Reply:
Because we don’t want it to do further check of whether this node was part of shard or is lagging behind. We return immediately from StartRetrieveFromHistory. It’s false only for the case in RECOVERY_ALL_SYNC when Zilliqa::Zilliqa() calls m_n.Install with false option.

Reply:
@ckyang Any side effect by replacing rejoiningAfterRecover?

Reply:
it was added for supporting rejoining of node that is not in shard after recovery from another testnet.